checks.d/elastic.py

```diff
@@ -424,12 +430,29 @@ def _get_data(self, url, config, send_sc=True):
         else:
             auth = None
 
+        # Load SSL configuration, if available.
+        # ssl_verify can be a bool or a string (http://docs.python-requests.org/en/latest/user/advanced/#ssl-cert-verification)
+        if isinstance(config.ssl_verify, bool):
```

Comment:
This could be rewritten more concisely in 
```python
if isinstance(config.ssl_verify, bool) or isinstance(config.ssl_verify, str):
    verify = config.ssl_verify
else:
    verify = None
```